controller/src/test/java/io/pravega/controller/server/eventProcessor/ScaleRequestHandlerTest.java

```diff
@@ -204,13 +204,15 @@ public void testScaleRequest() throws ExecutionException, InterruptedException {
         // And if someone changes retry durations and number of attempts in retry helper, it will impact this test's running time.
         // hence sending incorrect segmentsToSeal list which will result in a non retryable failure and this will fail immediately
         assertFalse(FutureHelpers.await(multiplexer.process(new ScaleOpEvent(scope, stream, Lists.newArrayList(6),
-                Lists.newArrayList(new AbstractMap.SimpleEntry<>(0.0, 1.0)), true, System.currentTimeMillis()))));
+                Lists.newArrayList(new AbstractMap.SimpleEntry<>(0.0, 1.0)), true, System.currentTimeMillis()),
```

Comment:
`Arrays.asList` is a built-in Java method that does the same thing. No need to use Guava here.